website/docs/cloud-docs/api-docs/account.mdx

```diff
@@ -70,6 +70,7 @@ curl \
     "attributes": {
       "username": "admin",
       "is-service-account": false,
+      "auth-type": "tfc",
```

Comment:
should these all be `auth-method`? That's the field mentioned in the message on the PR description, and querying the API seems to show the same:

```
...
"is-confirmed": true,
"is-sudo": false,
"auth-method": "tfc",
"has-linked-hcp": false,
"permissions": {
...
```

Reply:
Oh geeze! Thank you **so much** for catching that, Friday brain was silly and thought `auth-type` and `auth-method` were the same thing. 😨